spec/Overview.html

```diff
@@ -870,6 +881,85 @@ <h4>The getRandomValues method</h4>
               </p>
             </div>
           </section>
+          <section id="Crypto-method-randomUUID">
+            <h4>The randomUUID method</h4>
+            <p>
+              The <dfn id="dfn-Crypto-method-randomUUID" data-dfn-for=Crypto>randomUUID</dfn>
+              method generates a new <a data-cite="RFC4122#section-4.4">version 4 UUID</a>
+              and returns its <a data-cite="RFC2141#section-2.2">namespace specific string representation</a>
+              as described in <a data-cite="RFC4122#section-3">section 3</a> of [[RFC4122]].
+              To <dfn data-export="">generate a random UUID</dfn>:
+            </p>
+            <ol>
+              <li>
+                Let |bytes| be a [= byte sequence =] of length 16.
+              </li>
+              <li>
+                Fill |bytes| with cryptographically secure random bytes.
+              </li>
+              <li>
+                Set the 4 most significant bits of |bytes|[6], which represent the
+                UUID <a data-cite="RFC4122#section-4.1.3">version</a>, to `0100`.
+              </li>
+              <li>
+                Set the 2 most significant bits of |bytes|[8], which represent the
+                UUID <a data-cite="RFC4122#section-4.1.1">variant</a>, to `10`.
+              </li>
+              <li>
+                <p>
+                  Return the [= string/concatenate | string concatenation =] of «
+                </p>
+                <ol style="list-style-type: none">
+                  <li>
+                    [= hexadecimal representation =] of |bytes|[0], [= hexadecimal
+                    representation =] of |bytes|[1], [= hexadecimal representation =] of
+                    |bytes|[2], [= hexadecimal representation =] of |bytes|[3],
+                  </li>
+                  <li>
+                    `"-"`,
```

Comment:
Nit: the style we use is "`-`" (backticks inside) to separate the string value from the string syntax. This applies throughout.

(I also think that generally there are no spaces after `[=` and before `=]`, but I guess that's up to the editor as it's only visible in the source.)